### PR TITLE
Adding jcowie's coordinates.

### PR DESCRIPTION
**Pull request by jcowie**
@gitteacher what do you think

closes #940

### CODE REVIEW

**Comment by kayvee83** [comment]
@jcowie  File need to have the hypen before and after the content, otherwise the tests will fail. Just copy & paste the content from sample and update with your values.

**Comment by mkwoodburn** [comment]
Hello, great job starting in GitHub. @githubteacher

**Comment by brianamarie** [comment]
Hi @jcowie! Make sure there are no spaces at the beginning of each line. Let me know if I can help at all!

File: _pins/JCOWIE.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ ---
```
Comment (VTG38) [inline _pins/JCOWIE.json]: @jcowie 
This is the right one!

**Comment by crichID** [comment]
@jcowie I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/2531/commits/1caae66c61bf0cf4775e76ee264afca838f75cae. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by Friday (January 27, 2017), I will go ahead and do it just to keep the repository clean!

**Pull request merged**